crates/bevy_pbr/src/prepass/mod.rs

```diff
@@ -771,39 +787,90 @@ pub fn prepare_prepass_view_bind_group<M: Material>(
     }
 }
 
-pub fn queue_prepass_material_meshes<M: Material>(
-    (
-        opaque_draw_functions,
-        alpha_mask_draw_functions,
-        opaque_deferred_draw_functions,
-        alpha_mask_deferred_draw_functions,
-    ): (
-        Res<DrawFunctions<Opaque3dPrepass>>,
-        Res<DrawFunctions<AlphaMask3dPrepass>>,
-        Res<DrawFunctions<Opaque3dDeferred>>,
-        Res<DrawFunctions<AlphaMask3dDeferred>>,
-    ),
-    prepass_pipeline: Res<PrepassPipeline<M>>,
-    mut pipelines: ResMut<SpecializedMeshPipelines<PrepassPipeline<M>>>,
-    pipeline_cache: Res<PipelineCache>,
-    (render_meshes, render_mesh_instances): (
-        Res<RenderAssets<RenderMesh>>,
-        Res<RenderMeshInstances>,
-    ),
+#[derive(Resource)]
+pub struct SpecializedPrepassMaterialPipelineCache<M> {
+    map: HashMap<(MainEntity, MainEntity), (Tick, CachedRenderPipelineId), EntityHash>,
```

Comment:
Probably needs a comment explaining what the two `MainEntity`s are. I think the first one is view and the second one is object, right?